ci-operator/step-registry/ipi/conf/azuremag/ipi-conf-azuremag-ref.yaml

```diff
@@ -0,0 +1,41 @@
+ref:
+  as: ipi-conf-azuremag
+  from_image:
+   namespace: origin
+   name: centos
+   tag: '8'
+  commands: ipi-conf-azuremag-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  credentials:
```

Comment:
Per our discussion in slack, we can drop this custom credential secret to use the default credentials secret from ci namespace.